secure/key-manager/src/lib.rs

```diff
@@ -115,8 +116,15 @@ where
     /// initiate a key rotation when required. If something goes wrong, an error will be returned
     /// by this method, upon which the key manager will flag the error and stop execution.
     pub fn execute(&mut self) -> Result<(), Error> {
+        info!("The key manager has been created and is starting execution.");
         loop {
+            info!("The key manager is now re-checking the status of the keys..");
```

Comment:
No need for these prefixes, the logging will already say which module and line number, all you need to say is "Checking the status of the key." (note the double periods)

Reply:
Done, thanks!